packages/osv-offline/src/lib/osv-offline.ts

```diff
@@ -22,8 +25,8 @@ export class OsvOffline {
    * Asynchronously creates a new instance of {@link OsvOffline}
    * @returns A new instance of {@link OsvOffline}
    */
-  static async create(): Promise<OsvOffline> {
-    const instance = new OsvOffline();
+  static async create(githubToken?: string): Promise<OsvOffline> {
+    const instance = new OsvOffline(githubToken);
     await instance.initialize();
```

Comment:
```suggestion
    const instance = new OsvOffline();
     await instance.initialize(githubToken);
```
better pass directly to initialize, so we don't need to store it.